components/eventing-controller/pkg/backend/utils/eventmesh_utils.go

```diff
@@ -112,7 +113,7 @@ func getEventMeshEvents(typeInfos []EventTypeInfo, typeMatching eventingv1alpha2
 
 func ConvertKymaSubToEventMeshSub(subscription *eventingv1alpha2.Subscription, typeInfos []EventTypeInfo,
```

Comment:
The `ConvertKymaSubToEventMeshSub` sometimes returns an error, are there unit-tests to cover those scenarios?